astropy/wcs/utils.py

```diff
@@ -30,6 +38,20 @@
     "fit_wcs_from_points",
 ]
 
+WCSLIB_LT8 = Version(_wcs.__version__) < Version("8.0")
```

Comment:
Our nomenclature has been to use underscores on both sides of LT, i.e., `WCSLIB_LT_8`

Reply:
... I checked here
https://github.com/astropy/astropy/blob/98b399d52d730d91c2cd29babe1e8545caff219c/astropy/wcs/wcs.py#L106-L108
That's why I didn't put the underscore ... is it the wrong place?

Reply:
OK, no problem! I'm just used to how we do it with numpy...